src/time.rs

```diff
@@ -40,7 +40,11 @@ impl ClockId {
     }
 
     /// Sets time to `timespec` on the clock id
-    #[cfg(not(any(apple_targets, target_os = "redox", target_os = "hermit",)))]
+    #[cfg(not(any(
+        target_os = "macos",
```

Comment:
Did you not notice the `not` ?
```suggestion
    #[cfg(not(any(
        target_os = "ios",
        target_os = "tvos",
        target_os = "watchos",
```

Reply:
Oops, I am sorry I missed that, fixed!